clients/venice-push-job/src/main/java/com/linkedin/venice/heartbeat/DefaultPushJobHeartbeatSender.java

```diff
@@ -77,7 +77,7 @@ class DefaultPushJobHeartbeatSender implements PushJobHeartbeatSender {
   }
 
   // For test purpose.
-  VeniceWriter<byte[], byte[], byte[]> getVeniceWriter() {
+  public VeniceWriter<byte[], byte[], byte[]> getVeniceWriter() {
```

Comment:
Please share if there is any concerns here.

Reply:
Usually, the APIs which are intended for tests only are left package private on purpose, so as to avoid spreading their usage outside of test code. This is not a perfect blocker of usage outside of test code, but merely relies on the fact that most code happens to be in different packages, and therefore package-private will make the API unreachable.

In this particular case, IDK if it is a big deal for the API to graduate from intended-for-tests-only to generally/publicly available, but you could assess that and share your opinion. If there is no reason for making the API generally-available, then we would probably prefer keeping the package structure the same to avoid increasing visibility...

Reply:
The tests are rearranged and access is set to protected now.